server/parameters/CoreConfigFactory.java

```diff
@@ -42,11 +42,11 @@ static CoreConfig config(CoreConfigParser parser) {
         return config(new HashSet<>(), parser);
     }
 
-    static CoreConfig config(Set<Option> overrides, CoreConfigParser parser) {
+    public static CoreConfig config(Set<Option> overrides, CoreConfigParser parser) {
         return config(CONFIG_PATH, overrides, parser);
     }
 
-    static CoreConfig config(Path file, Set<Option> overrides, CoreConfigParser parser) {
+    public static CoreConfig config(Path file, Set<Option> overrides, CoreConfigParser parser) {
```

Comment:
Make public, in order to allow reading a specific config file and add overrides from tests